src/e2e/kotlin/fi/aalto/cs/apluscourses/e2e/steps/CommonSteps.kt

```diff
@@ -2,78 +2,47 @@ package fi.aalto.cs.apluscourses.e2e.steps
 
 import com.intellij.remoterobot.RemoteRobot
 import com.intellij.remoterobot.stepsProcessing.step
+import com.intellij.remoterobot.utils.attempt
 import fi.aalto.cs.apluscourses.e2e.fixtures.customComboBox
 import fi.aalto.cs.apluscourses.e2e.fixtures.dialog
 import fi.aalto.cs.apluscourses.e2e.fixtures.heavyWeightWindow
 import fi.aalto.cs.apluscourses.e2e.fixtures.welcomeFrame
-import okhttp3.OkHttpClient
-import okhttp3.Request
-import java.io.File
 import java.time.Duration
 
 class CommonSteps(val remoteRobot: RemoteRobot) {
 
-  fun createProject() {
-    with (remoteRobot) {
-      step("Create New Empty Project") {
-        with(welcomeFrame()) {
-          newProjectButton().click()
-          with(dialog("New Project")) {
-            findText("Empty Project").click()
-            button("Next").click()
-            button("Finish").click()
-          }
-        }
-      }
-      step("Add OpenJDK 11") {
-        with(dialog("Project Structure", Duration.ofSeconds(20))) {
-          sidePanel().findText("Project").click()
-          customComboBox("\u001BProject SDK:").dropdown()
-          with(heavyWeightWindow()) {
-            findText("Add SDK").click()
-            HierarchyDownloader.catchHierarchy {
-              heavyWeightWindow().findText("Download JDK...").click()
+    fun createProject() {
+        with(remoteRobot) {
+            step("Create New Empty Project") {
+                with(welcomeFrame()) {
+                    newProjectButton().click()
+                    with(dialog("New Project")) {
+                        findText("Empty Project").click()
+                        button("Next").click()
+                        button("Finish").click()
+                    }
+                }
+            }
+            step("Add OpenJDK 11") {
+                with(dialog("Project Structure", Duration.ofSeconds(20))) {
+                    sidePanel().findText("Project").click()
+                    customComboBox("\u001BProject SDK:").dropdown()
+                    with(heavyWeightWindow()) {
+                        attempt(2) {
```

Comment:
If this is a common problem with comboboxes/heavyweightwindows, `attempt(2)` would be better to be put "under the hood" so it doesn't clutter test procedures

Reply:
is there a possibility it does the first click successfully but not the second one, and then on the second attempt it closes the thing it opened? maybe it should be 3 attempts in that case

Reply:
I don't think it executes twice if successful on the first try. After all, it's the same code executed across all the environments (Linux, Mac), so if the assumption would hold, then for Linux it would open the window, close the window, fail.